backend/backend.go

```diff
@@ -162,10 +165,15 @@ func NewBackend(arguments *arguments.Arguments, environment Environment) (*Backe
 		return nil, err
 	}
 	backend.notifier = notifier
+	backend.socksProxy = socksproxy.NewSocksProxy(backend.config.AppConfig().Backend.UseProxy, backend.config.AppConfig().Backend.ProxyAddress)
+	backend.log.Println("Creating updater instance")
```

Comment:
`Println()` -> `Info()`, here and in more places. We do not append newlines anywhere else (I think this is automatically done), and `Info` is more descriptive.